addons/web/static/src/scss/translationdialog.scss

```diff
@@ -0,0 +1,12 @@
+// translation dialog
+.translation_dialog {
+    & .language_code,
+    & .language {
+        font-weight: bold;
+    }
+
+     .row {
+        margin-bottom: 5px;
```

Comment:
i'd set at least 10px of padding so that it's also looking good in mobile (cf my requested change in the template)

Reply:
9, but not more 😉